bands/templates/bands/stage_detail.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+
+<h1>{{stage.name}}</h1>
+
+
+<h2> Bands </h2>
```

Comment:
It's not clear from this heading that this is showing the bands that have or are going to perform on this stage. The text should be expanded to fix that.